src/momento-redis-adapter.ts

```diff
@@ -824,6 +830,14 @@ export class MomentoRedisAdapter
       return 'OK';
     }
   }
+
+  pipeline(commands?: Command[][]): ChainableCommander {
+    const pipeline = new Pipeline(this);
+    if (Array.isArray(commands)) {
```

Comment:
If `commands` isn't an array, do we want to silently succeed with no commands in the pipeline vs throw an error? Let's also document this line here is for runtime type safety and mirrors the ioredis implementation.